### ISSUE TITLE
Add handling connection refused error

### ISSUE THREAD

**Issue by wojtii** [nordcloud/mfacli#7]
I have encountered error:
```
Error: dial unix ~/.mfacli/mfacli.sock: connect: connection refused
```

It was probably caused by os update.
The solution was to remove `mfacli.sock` file.

We could handle this error automatically or add solution to readme.

**Comment by kevwargo** [comment]
Yep, I'm aware of the issue, will work on it :slightly_smiling_face:
